### PR TITLE
Add python-dev to Dockerfile to fix python2.7

### PR DESCRIPTION
**Pull request by stephendonner**
@hackebrot @jrbenny35 @davehunt r, thoughts?

### CODE REVIEW

**Comment by stephendonner** [comment]
Actually, I think perhaps that https://github.com/mozilla/fxapom/pull/262 is what we need, based on https://travis-ci.org/mozilla/fxapom/jobs/414393991

**Comment by davehunt** [comment]
It's not clear from the summary or description what issue this is fixing. Could you add more context @stephendonner?

**Comment by stephendonner** [comment]
@davehunt yes, sorry; this failure here: https://travis-ci.org/mozilla/fxapom/jobs/413222814#L2586

**Comment by davehunt** [comment]
> yes, sorry; this failure here: https://travis-ci.org/mozilla/fxapom/jobs/413222814#L2586

Right, I see now how they're related. Thanks.

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ RUN apt-get update \
   && apt-get install -y software-properties-common \
   && add-apt-repository ppa:deadsnakes/ppa \
   && apt-get update \
-  && apt-get install -y bzip2 curl firefox git python2.7 python3.6 python3-pip \
+  && apt-get install -y bzip2 curl firefox git python2.7 python-dev python3.6 python3-pip \
```
Comment (m8ttyB) [inline Dockerfile]: beauty

**Pull request merged**